Dockerfile.testbase

```diff
@@ -2,12 +2,16 @@ FROM python:3.7
 LABEL MAINTAINER=dotkom@online.ntnu.no
 
 # Install deps
+COPY poetry.lock pyproject.toml /
+
 RUN curl -sL https://deb.nodesource.com/setup_8.x | bash - && \
     apt-get remove -y curl && apt-get install -y --no-install-recommends \
     nodejs libjpeg-dev ghostscript && \
     npm install -g less && \
-    npm install -g yarn && \
-    pip install tox
```

Comment:
I don't get why this is installing tox, isn't that done from `.drone.yml` anyway?